datajoint/expression.py

```diff
@@ -931,3 +956,69 @@ def aggr(self, group, **named_attributes):
         )
 
     aggregate = aggr  # alias for aggr
+
+
+def _flatten_attribute_list(primary_key, attrs):
+    """
+    :param primary_key: list of attributes in primary key
+    :param attrs: list of attribute names, which may include "KEY", "KEY DESC" or "KEY ASC"
+    :return: generator of attributes where "KEY" is replaced with its component attributes
+    """
+    sql_keywords = [
```

Comment:
I wouldn't track all keywords. Just enclose all attribute names in backquotes.

Reply:
This works for MySQL and we already use this approach for declaring tables and forming queries. See how the WHERE clause is formed from dict resitrctions.

Reply:
Using a similar method as how the WHERE clause is built wasn't working because it would wrap "asc" and "desc" too if it was included, which raises a "Column not found" error.